### PR TITLE
Support to return error codes from Metaclient 

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: src/common/clients/graph/GraphClient.cpp [hunk 1]
```diff
@@ -28,8 +28,8 @@ GraphClient::~GraphClient() {
 }
 
 
-cpp2::ErrorCode GraphClient::connect(const std::string& username,
-                                     const std::string& password) {
+nebula::cpp2::ErrorCode GraphClient::connect(const std::string& username,
+                                             const std::string& password) {
```
Comment (monadbobo) [inline src/common/clients/graph/GraphClient.cpp:32]: space？

File: src/common/clients/graph/GraphClient.cpp [hunk 2]
```diff
@@ -74,18 +74,19 @@ void GraphClient::disconnect() {
 }
 
 
-cpp2::ErrorCode GraphClient::execute(folly::StringPiece stmt,
-                                     cpp2::ExecutionResponse& resp) {
+nebula::cpp2::ErrorCode GraphClient::execute(folly::StringPiece stmt,
+                                             cpp2::ExecutionResponse& resp) {
```
Comment (monadbobo) [inline src/common/clients/graph/GraphClient.cpp:78]: space？

File: src/common/interface/common.thrift [hunk 3]
```diff
@@ -180,3 +180,154 @@ struct KeyValue {
...
+    // 2xxxx, for graph Failure
+    // Authentication error
+    E_USERNAME_NOT_FOUND          = 20001,
+
+    E_SESSION_NOT_EXIST           = 20002,
+    E_SESSION_EXPIRED             = 20003,
+
+    E_SYNTAX_ERROR                = 20004,
+    // Nothing is executed When command is comment
+    E_STATEMENT_EMTPY             = 20005,
```
Comment (monadbobo) [inline src/common/interface/common.thrift:204]: Do we need to distinguish between storage and meta errors?

**Comment by panda-sheep** [comment]
Please resolve the conflict?

**Pull request closed without merging**